src/casp/bq_scripts/anndata_to_avro.py

```diff
@@ -95,10 +112,6 @@ def dump_cell_info(adata, filename, cas_cell_index_start, ingest_id):
     adata.obs["original_cell_id"] = adata.obs.index
     adata.obs["cas_cell_index"] = np.arange(cas_cell_index_start, cas_cell_index_start + len(adata.obs))
 
-    row_index_to_cas_cell_index = [None] * len(adata.obs)
-    for row in range(0, len(adata.obs)):
-        row_index_to_cas_cell_index[row] = adata.obs["cas_cell_index"].iloc[[row]][0]
-
     schema = {
```

Comment:
Refactor this schema to a more appropriate place, maybe under `/bq_scripts/schemas`?